lib/views/pre-onboarding_view.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:actualia/utils/themes.dart';
+import 'package:actualia/views/interests_wizard_view.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:introduction_screen/introduction_screen.dart';
+
+class PreOnBoardingPage extends StatefulWidget {
+  const PreOnBoardingPage({super.key});
+
+  @override
+  PreOnBoardingPageState createState() => PreOnBoardingPageState();
+}
+
+class PreOnBoardingPageState extends State<PreOnBoardingPage> {
+  final introKey = GlobalKey<IntroductionScreenState>();
+
+  void _onIntroEnd(context) {
+    Navigator.of(context).pushReplacement(
+      MaterialPageRoute(
+          builder: (_) =>
+              const InterestWizardView()), // For testing purposes, we can replace this with HomePage()
+    );
+  }
+
+  Widget _buildImage(String assetName, [double width = 250]) {
+    return Image.asset('assets/img/$assetName', width: width);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    const titleSmall = TextStyle(
+        fontFamily: "EB Garamond",
+        fontSize: 14.0,
+        height: 1.4,
+        fontWeight: FontWeight.w300);
+    const bodyTextStyle = TextStyle(
+        fontFamily: "Fira Code",
+        fontSize: 14.0,
+        height: 1.4,
+        fontWeight: FontWeight.w300);
+
+    const pageDecoration = PageDecoration(
+      titleTextStyle: titleSmall,
+      bodyTextStyle: bodyTextStyle,
+      bodyPadding: EdgeInsets.fromLTRB(16.0, 0.0, 16.0, 16.0),
+      pageColor: THEME_LIGHTGRAY,
+      imagePadding: EdgeInsets.zero,
+    );
```

Comment:
```suggestion
var pageDecoration = PageDecoration(
      titleTextStyle: Theme.of(context).textTheme.titleMedium!,
      bodyTextStyle: Theme.of(context).textTheme.bodyMedium!,
      bodyPadding: const EdgeInsets.fromLTRB(16.0, 0.0, 16.0, 16.0),
      pageColor: THEME_LIGHTGRAY,
      imagePadding: EdgeInsets.zero,
    );
```

This seems to do the trick ;)

Reply:
Wtf it didn't compile when I tried this method. Seems to work now, thanks!